tests/yam/validate/validate_minimal_system.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Validate the base product from /etc/products.d/baseproduct.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use testapi;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal();
+
+    my @installed_packages = split /\n/, script_output('rpm -qa');
+    my %is_installed = map { $_ => 1 } @installed_pkgs;
+
+    record_info("Package number:", scalar @installed_pkgs);
```

Comment:
Number of package (due to package number might be understood like an specific package id).

Reply:
you got `0` because you used a different name: `installed_pkgs`
https://openqa.suse.de/tests/18972097#s

Reply:
Yes. Changed.

> Number of package (due to package number might be understood like an specific package id). 'Package count' also works to shorten the string.

Changed to `Package count`